addons/mail/static/src/components/autocomplete_input/autocomplete_input.xml

```diff
@@ -2,7 +2,7 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.AutocompleteInput" owl="1">
-        <input class="o_AutocompleteInput" t-attf-class="{{ className }}" t-on-blur="_onBlur" t-on-focusin="props.onFocusin" t-on-keydown="_onKeydown" t-att-placeholder="props.placeholder" t-ref="root"/>
+        <input class="o_AutocompleteInput" t-attf-class="{{ className }}" t-on-blur="_onBlur" t-on-focusin="_onFocusin" t-on-keydown="_onKeydown" t-att-placeholder="autocompleteInputView and autocompleteInputView.placeholder" t-ref="root"/>
```

Comment:
Don't have much choice in defining local handlers.
Can't guard handlers in models, and top-level `t-if` is problematic due to implementation detail of relying on `ui-autocomplete` and using component lifecycle to cleanly destroy `ui-autocomplete` stuffs on the root node.
Top-level `t-if` would make test fail, due to `ui-autocomplete` polluting the DOM at end of test.